### PR TITLE
Update README.md

### LINKED ISSUE #52: testing linux support for yast
## Description
We believe that `yast` supports linux environments because the same FileSystem API works on both Linux and MacOS. `yast` has been tested and vetted for properly working on macOS. But we need to test the same for Linux OSes such as Ubuntu, Fedora, Debian based distros. 
If in some environment, it does not work? Is there a way to fix it? Do we need to make some changes to the code or provide additional documentation instructions/installation instructions for different environments? 

## Are you raising this issue under hacktoberfest? 
Yes, I'm raising this issue under hacktoberfest 2022. This is a beginner issue, should only require small code changes and some documentation changes.

### PR DESCRIPTION
**Pull request by moazfarrukh**
added helpful side notes for ubuntu users during installation
also told what to do in case the movie title has multiple words.
closes #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 ```bash
 npm install webtorrent-cli -g
 ```
+> you might need to prefix this with sudo for some linux distros
```
Comment (qascade) [inline README.md:18]: Change this line to `Note: use sudo, if required.`

File: README.md [hunk 2]
```diff
@@ -27,6 +28,7 @@ git clone https://github.com/qascade/yast && cd yast
 ```bash
 go build ./...
 ```
+>try `go build` if `go build ./...` does not work
```
Comment (qascade) [inline README.md:31]: Remove this line and replace `go build ./...` with `go build`

File: README.md [hunk 3]
```diff
@@ -60,6 +62,7 @@ Use "yast [command] --help" for more information about a command.
 ```bash
 ./yast search -m spiderman
 ```
+> if the movie name contains multiple words type it inside double quotes  e.g. "spiderman no way home"
```
Comment (qascade) [inline README.md:65]: Do the following changes here :
1. `if` -> `If`

2.`multiple words` -> `multiple words separated by spaces`

3.`type it` -> `wrap it ` 

4. write the full command in eg.

**Review by qascade — CHANGES_REQUESTED** [review]
Please do the requested small changes.

File: README.md (hunk 3 above)
Comment (moazfarrukh) [reply README.md]: @qascade made a new commit. have resolved these :)

**Review by bayaosborn — APPROVED** [review]
Ultimate.

**Review by qascade — APPROVED** [review]
LGTM!

**Pull request merged**